pypeit/core/datacube.py

```diff
@@ -795,6 +795,8 @@ def make_whitelight_frompixels(all_ra, all_dec, all_wave, all_sci, all_wghts, al
             Number of RA spaxels in the reference white light image
         numdec (int, optional):
             Number of DEC spaxels in the reference white light image
+        trim (int, optional):
```

Comment:
Please try to indicate shapes of arrays in the docs for both inputs and outputs.